src/Cardano/Prelude/Orphans.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE CPP               #-}
```

Comment:
And why do we need that?

Reply:
Use some CPP previously, but realised that I didn't need to.  Removed.  Thanks!